docs/guides/02-using-style-helpers.md

```diff
@@ -0,0 +1,231 @@
+## Using style helpers
+
+In this guide, you will learn how to create insightful visualizations with few lines of code using the Web SDK style helpers. It is recommended to have read the Getting Started guide to understand the basic structure of a Web SDK application.
```

Comment:
How about adding somewhere a note like this?

"Web SDK uses internally deck.gl, so you can always use pure deck.gl styles if you prefer; see for example this layer style here (TODO: links to deck.gl style example / deck.gl GeoJSON reference). But with the style helpers you can get more easily some common layer visualizations (like for example creating a bubble map taking into account different classification methods for your bins)"

Reply:
Added